Dockerfile.test

```diff
@@ -1,17 +1,16 @@
 FROM ubuntu:18.04
 
-RUN apt-get update
-RUN apt-get install -y python3 python3-pip
-RUN apt-get install -y vim
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip vim bc
 
 # Install Dependencies
+RUN ln -s /usr/bin/pip3 /usr/bin/pip
```

Comment:
This seems unsafe, the better way to ensure you always get the right pip is to do:

python3 -m pip [command]

Reply:
Resolved this by changing the run_tests.sh